Scala/Scala.sublime-syntax

```diff
@@ -610,6 +708,43 @@ contexts:
     - match: '(?=\S)'
       pop: true
 
+  function-parameter-list-contents:
+    - match: '\)'
+      scope: punctuation.section.group.end.scala
+      pop: true
+    - match: '\b(?:using|implicit)\b'
+      scope: storage.modifier.other.scala
+    - match: '({{alphaid}})(?=\s*:)'
+      captures:
+        1: variable.parameter.scala
+      push:
+        - match: '{{typeprefix}}'
+          captures:
+            1: punctuation.ascription.scala
+          set:
+            - match: '(\*)\s*(?=[\),=])'
+              captures:
+                1: keyword.operator.varargs.scala
+            - match: '(?=\))'
+              pop: true
+            - match: ','
+              scope: punctuation.separator.scala
+              pop: true
+            - include: delimited-type-expression
+            - match: '(?=[=])'
```

Comment:
Character class shouldn't be required to match single `=` only.

```suggestion
            - match: '(?==)'
```

Reply:
This is old code that got moved around, but I *think* I did it because `(?==)` is really hard to read when you have a ligature font (since `==` renders like a long `=`).

Reply:
Do you have PackageDev installed for highlighting? I think the different scopes for `?=` and `=` should prevent collapse into a ligature.

Reply:
I think I removed it at some point because it was annoying me relative to the base yaml mode. If there's a strong preference for removing the spurious character class, I don't mind merging the suggestion.

Reply:
There's not. It is not of any functional relevance.